static/styles/subscriptions.scss

```diff
@@ -590,167 +590,171 @@ form#add_new_subscription {
-}
-
-.stream-row .check {
-    width: 25px;
-    height: 25px;
-    position: relative;
-    margin-right: 8px;
-    margin-top: 9px;
-    background-size: 60% auto;
-    background-repeat: no-repeat;
-    background-position: center center;
-}
-
-.stream-row .check svg {
-    fill: transparent;
-    width: 70%;
-    margin: 0% 15%;
-}
-
-.stream-row .checked svg {
-    fill: hsl(170, 48%, 54%);
-}
-
-.stream-row .check.checked:hover svg {
-    opacity: 0.5;
-}
-
-.stream-row .icon {
-    width: 35px;
-    height: 35px;
-    margin-right: 8px;
-    margin-top: 4px;
-    background-color: hsl(300, 100%, 25%);
-    border-radius: 4px;
-    color: hsl(0, 0%, 100%);
-}
-
-.stream-row .icon .symbol {
-    font-weight: 600;
-    font-size: 1.1em;
-}
-
-.stream-row .icon .fa-lock {
-    font-size: 1.4em;
-}
-
-.stream-row .icon .hashtag {
-    font-size: 1.4em;
-    font-weight: 600;
-}
-
-.stream-row .icon .hashtag::-moz-selection {
-    background-color: transparent;
-}
-
-.stream-row .icon .hashtag::selection {
-    background-color: transparent;
-}
-
-.stream-row .sub-info-box {
-    width: calc(100% - 90px);
-}
-
-.stream-row .sub-info-box .top-bar,
-.stream-row .sub-info-box .bottom-bar {
-    display: flex;
-    justify-content: space-between;
-    position: relative;
-}
-
-.stream-row .sub-info-box .top-bar > div {
-    display: inline-block;
-    vertical-align: top;
-}
-
-.stream-row .sub-info-box .top-bar .stream-name {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-}
-
-.stream-row .sub-info-box .top-bar .stream-name {
-    font-weight: 600;
-}
-
-.stream-row .sub-info-box .top-bar .subscriber-count {
-    white-space: nowrap;
-    color: hsl(0, 0%, 67%);
-}
-
-.stream-row .sub-info-box .top-bar .subscriber-count-text,
-.stream-row .sub-info-box .top-bar .subscriber-count-lock {
-    margin-right: 5px;
-}
-
-.stream-row .sub-info-box .bottom-bar {
-    margin-top: 2px;
-    line-height: 1.5;
-}
-
-.stream-row .sub-info-box .bottom-bar > div {
-    display: inline-block;
-    vertical-align: bottom;
-}
 
-.stream-row .sub-info-box .bottom-bar .description {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-}
-
-.stream-row .sub-info-box .description:empty::after {
-    content: attr(data-no-description);
-    font-style: italic;
-    color: hsl(0, 0%, 67%);
-}
-
-.stream-row .sub-info-box .bottom-bar .stream-message-count {
-    white-space: nowrap;
-    color: hsl(0, 0%, 67%);
-}
-
-.stream-row .sub-info-box .bottom-bar .stream-message-count-text {
-    margin-right: 5px;
-}
+    .check {
+        width: 25px;
+        height: 25px;
+        position: relative;
+        margin-right: 8px;
+        margin-top: 9px;
+        background-size: 60% auto;
+        background-repeat: no-repeat;
+        background-position: center center;
+
+        svg {
+            fill: transparent;
+            width: 70%;
+            margin: 0% 15%;
+        }
+
+        &.checked {
```

Comment:
I think at least this use of  `&` is buggy, see https://css-tricks.com/the-sass-ampersand/.  The original rule here was 
`.stream-row .checked svg`, not `.stream-row.checked svg` which is what you have here...

Reply:
No, I believe I didn't make a mistake here. The original CSS was:
```css
.stream-row .check svg {
    fill: transparent;
    width: 70%;
    margin: 0% 15%;
}

.stream-row .checked svg {
    fill: hsl(170, 48%, 54%);
}

.stream-row .check.checked:hover svg {
    opacity: 0.5;
}
```

The DOM looks like:
```html
<div class="stream-row" data-stream-id="1" data-stream-name="announce">
    <div class="check checked sub_unsub_button">
```
So, really, `.stream-row .checked svg` should be `.stream-row .check.checked svg`. Also, `&` refers to the parent selector of the current element, which is `.check` and not `.stream-row` since it's nested under the `.check` selector which is nested under `.stream-row`.

Reply:
Yes, but if I'm not mistaken, `&.checked` is how you generate `.stream-row.checked`, not `.stream-row .checked` - i.e. expecting the checked to be on the same element as the stream-row.